test/integration/sanity/proxy-http.test.js

```diff
@@ -1,13 +1,18 @@
 describe('sanity test', function () {
-    var _ = require('lodash'),
+    var ProxyConfigList = require('postman-collection').ProxyConfigList,
+        _ = require('lodash'),
         proxy = require('http-proxy'),
 
         server,
         testrun;
 
     before(function (done) {
         var port = 9090,
-            proxyList = [{match: '*://*.getpostman.com/*', server: 'http://localhost:' + port, tunnel: false}];
+            proxyList = new ProxyConfigList({}, [{
```

Comment:
@vjrngn you need to pass the proxyList as sdk Instance instead of a normal ListArray.